src/SBP_coefficients/MattssonAlmquistVanDerWeide2018Minimal.jl

```diff
@@ -67,7 +71,7 @@ function Base.getindex(grid::BoundaryAdaptedGrid, i::Int)
 end
 
 Base.size(grid::BoundaryAdaptedGrid) = (length(grid),)
-Base.step(grid::BoundaryAdaptedGrid) = step(grid.uniform_grid)
+Base.step(grid::BoundaryAdaptedGrid) = (grid.xmax - grid.xmin - 2*grid.xstart[end]) / (length(grid) + 1 - 2*length(grid.xstart))
```

Comment:
Would it make sense to store $\Delta x$ instead in the `BoundaryAdaptedGrid`?

Reply:
Yes, makes sense ([50c75bd](https://github.com/ranocha/SummationByPartsOperators.jl/pull/281/commits/50c75bd7b2e101a249eec709de5cf2f44e79dc23)).